control/preimage/src/helpers.rs

```diff
@@ -101,3 +184,71 @@ pub async fn query_para_id(api: &OnlineClient<PolkadotConfig>) -> Result<u32, Bo
 
     Ok(bridge_hub_para_id.0)
 }
+
+pub async fn query_para_id_asset_hub(
+    api: &OnlineClient<PolkadotConfig>,
+) -> Result<u32, Box<dyn std::error::Error>> {
+    let storage_query = asset_hub_runtime::storage().parachain_info().parachain_id();
+    let asset_hub_para_id = api
+        .storage()
+        .at_latest()
+        .await?
+        .fetch(&storage_query)
+        .await?
+        .expect("parachain id not set");
+
+    Ok(asset_hub_para_id.0)
+}
+
+pub fn utility_batch(calls: Vec<RelayRuntimeCall>) -> RelayRuntimeCall {
+    RelayRuntimeCall::Utility(
+        crate::relay_runtime::runtime_types::pallet_utility::pallet::Call::batch { calls },
```

Comment:
Is the [batch_all](https://github.com/paritytech/polkadot-sdk/blob/994003854363b3c5fd0d7343f93aa1e54edb1ad0/substrate/frame/utility/src/lib.rs#L323) make more sense here?

Reply:
Turns out they use `force_batch` for OpenGov proposals, so I changed to that.